portafly/src/tests/components/pages/accounts/AccountsTable.test.tsx

```diff
@@ -48,202 +53,22 @@ afterEach(resetMocks)
 
 const setup = () => {
   const wrapper = render(<AccountsTable />)
-  const getRow = (account: IDeveloperAccount) => wrapper.getByText(account.org_name).closest('tr') as HTMLElement
-  const bulkSelectorDropdown = within(wrapper.container.querySelector('#data-list-bulk-selector-dropdown') as HTMLElement)
-  const bulkActionsDropdown = within(wrapper.container.querySelector('#data-list-bulk-actions-dropdown') as HTMLElement)
+  const table = within(wrapper.queryByRole('grid') as HTMLElement)
+  const getRow = (account: IDeveloperAccount) => table.getByText(account.org_name).closest('tr') as HTMLElement
 
   return {
     ...wrapper,
-    getRow,
-    bulkSelectorDropdown,
-    bulkActionsDropdown
+    getRow
   }
 }
-
-it('should be able to sort columns', () => {
-  const { container } = setup()
-  const table = within(container.querySelector('table') as HTMLElement)
-  fireEvent.click(table.getByText(/accounts_table.group_header/))
-
-  expect(defaultTable.setSortBy).toHaveBeenLastCalledWith(1, 'asc', true)
+it('should render the data list table with the accounts provided', () => {
+  const { getByText, getRow } = setup()
+  developerAccounts.map((account) => expect(getRow(account)).toBeInTheDocument())
+  expect(() => getByText(/accounts_table.empty_state/)).toThrowError()
```

Comment:
Just an alternative:
```suggestion
  expect(() => queryByText(/accounts_table.empty_state/)).not.toBeInTheDocument()
```